supporting_content/dimensions/chargedescription.md

```diff
@@ -0,0 +1,33 @@
+# Column: ChargeDescription
+
+## Example provider mappings
+
+Current resource types found or extracted from available data sets:
+
+| Provider  | Data set                | Column                                            	|
+| :-------- | :---------------------- | :-------------------------------------------------------|
+| AWS       | CUR                     | lineItem/LineItemDescription 				|
+| GCP       | BigQuery Billing Export | sku.description                                         |
+| Microsoft | Cost details            | Not available      			                |
+
+## Example usage scenarios
+
+| Provider  | Data set                | Scenario                           | Value                    |
+|:----------|:------------------------|:-----------------------------------|:-------------------------|
+| AWS       | CUR                     | Not available                      | $0.00 per GB - US West (Oregon) data transfer from US West (Northern California) |
+| GCP       | BigQuery Billing Export | Not available                      | Not Available            |
+| Microsoft | Cost details            | via Cost export file		   | Not Available            |
+
+## Discussion Topics
+
+### Minimum Information Requirements vs Open-Ended
+
+We deliberated if we should be more specific and define the minimum required information for this column?
+
+- A list of FOCUS columns which we believe provide this high-level context without the need for additional discovery and thus must be included (concatenated) as part of the Charge description? (Irena)
```

Comment:
I would venture that the charge description should be unambiguous once combined with the  product name / service description information.

Reply:
I think we need to be explicit here at least at a conceptual level. I'm fine if we don't give the algorithm for concatenation, but we need to be clear for providers who don't do this today. If we aren't, it's going to be wildly inconsistent and ultimately useless. What happens when we get a paragraph description of marketing content here? Is that okay? I'm assuming not.

Reply:
But, we can't think of this from just a cloud perspective.  From the SaaS angle, this column is meant to act as the part of the charge that gives the most context around what is being billed.  Most of the time, from an invoice or billing/usage summary from SaaS products, these details don't exist as columns to concatenate.

Reply:
I know why you would say that! 

But I want to fight a little harder for CSP applicability. Remember we're getting articles written about us because of the applicability to public cloud. We cannot and should not walk away from that..

I am happy to address the SaaS case additionally in this copy, but I think it's super important to make sure things are super clear for current and future cloud-provider-type services.

Reply:
added suggestion above. thanks for prodding